src/util/OAuthErrors.ts

```diff
@@ -0,0 +1,130 @@
+import { AuthSdkError, FieldError, OAuthError as SdkOAuthError} from '@okta/okta-auth-js';
+import * as Errors from './Errors';
```

Comment:
@denysoblohin-okta has an approved PR which changes this file to expose named exports. Before merging this PR, I recommend waiting for PR https://github.com/okta/okta-signin-widget/pull/2763 to be merged, then update and resolve the conflict.